### PR TITLE
Feature/improve test coverage Closes #6

### LINKED ISSUE #6: Add more test coverage to current code base
Currently only utils and config have tests, and even those are minimal. Not sure how we could test the commands, but we could test the directions and weather packages.

### PR DESCRIPTION
**Pull request by marioharper**
@johnstamatakos do you think this adds enough coverage? We should regression test before merging also. Since I had to make some changes to make some stuff testable.

### CODE REVIEW

File: cmd/utils/utils_test.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+	"github.com/PennTex/commuter/cmd/utils"
+)
+
+// cant test this because process error exists the process
+func ExampleProcessError() {
+	err := errors.New("Test")
+	es := "Error String"
+
+	utils.ProcessError(err, es)
```
Comment (johnstamatakos) [inline cmd/utils/utils_test.go]: Should we make the utils tests more robust/in line with the other testing standards?

File: weather/dark_sky.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
+package weather
```
Comment (johnstamatakos) [inline weather/dark_sky.go:1]: Should we add a weather test suite for consistency?

**Review by johnstamatakos — COMMENTED** [review]
- [ ] Add make test command
- [ ] Running commuter without config file (such as a new user) has ugly panic error - could be from refactor
- [ ] Running commuter init as a new user (w/o config file) gives error: `Getting location by name: Location work not found.`  - could be from refactor
- [ ] No test files for commuter root, cmd folder, and weather (need to decide how much coverage we want)
- [ ] Test output in console kinda sucks, but I don't know if we can improve that

**Comment by johnstamatakos** [comment]
Otherwise, looks great dude.  Love the interfaces and pulling out maps/dark sky was a good idea.

File: weather/dark_sky.go (hunk 2 above)
Comment (marioharper) [reply weather/dark_sky.go:1]: honestly didn't know what to test here

**Pull request merged**